### PR TITLE
Reconcile Enterprise and OSS release notes

### PR DESCRIPTION
**Pull request by pierwill**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -15,35 +15,81 @@ The release of InfluxDB Enterprise 1.9 is different from previous InfluxDB Enter
...
 ### Features
 - Support user-defined *node labels*.
   Node labels let you assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
 - Improve performance of `SHOW SERIES CARDINALITY` and `SHOW SERIES CARDINALITY from <measurement>` InfluxQL queries.
   These queries now return a `cardinality estimation` column header where before they returned `count`.
-- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
+- Improve diagnostics for license problems.
+  Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
 - Add improved [ingress metrics](/enterprise_influxdb/v1.9/administration/config-data-nodes/#ingress-metric-by-measurement-enabled--false) to track points written by measurement and by login.
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:28]: allows for collection of statistics of points, values AND new series written per measurement and by login.

Previously we had points and values.  But the new series has been added...along with the measurement and login options.  Further...this data is collected at the data node level.  These metrics are exposed for each data node and operators can use these to aggregate the write requests across the entire cluster to gain a detailed understanding of the growth of series within a measurement and what user credentials are being used to write data to the cluster and more!

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -15,35 +15,81 @@ The release of InfluxDB Enterprise 1.9 is different from previous InfluxDB Enter
...
 - Upgrade to [Flux v0.113.0](/influxdb/cloud/reference/release-notes/flux/#v01130-2021-04-21).
 
+
+
+
+- feat(prometheus): update prometheus remote protocol <!-- https://github.com/influxdata/influxdb/pull/17814 -->
+- feat: meancount aggregation for WindowAggregate pushdown in enterprise <!-- https://github.com/influxdata/influxdb/pull/21291 -->
+- feat(tsi): optimize series iteration <!-- https://github.com/influxdata/influxdb/pull/20544 -->
+- fix: group by returns multiple results per group in some circumstances <!-- https://github.com/influxdata/influxdb/pull/21631 -->
+- feat: add arguments to flux to function <!-- https://github.com/influxdata/influxdb/pull/20873 -->
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:40]: we need to back tick `to`

Comment (timhallinflux) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:40]: as in Flux `to` function

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 3]
```diff
@@ -15,35 +15,81 @@ The release of InfluxDB Enterprise 1.9 is different from previous InfluxDB Enter
...
+- feat: Add WITH KEY to show tag keys <!-- https://github.com/influxdata/influxdb/pull/20793 -->
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:41]: can you check to see that this appears in the InfluxQL spec too?  /enterprise_influxdb/v1.9/query_language/explore-schema/#show-tag-keys

you can borrow the WITH syntax from:
enterprise_influxdb/v1.9/query_language/explore-schema/#show-tag-values

example: SHOW TAG KEYS FROM cpu WITH KEY IN (host, region)

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 4]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
+  Allow for collection of statistics of points, values, and new series written per measurement and by login.
+  This data is collected and exposed at the data node level.
+  With these metrics you can, for example:
+  aggregate the write requests across the entire cluster,
+  monitor the growth of series within a measurement,
+  and track what user credentials are being used to write data.
+- Support authentication for Kapacitor via LDAP.
 - Support for [configuring Flux query resource usage](/enterprise_influxdb/v1.9/administration/config-data-nodes/#flux-controller) (concurrency, memory, etc.).
 - Upgrade to [Flux v0.113.0](/influxdb/cloud/reference/release-notes/flux/#v01130-2021-04-21).
+- Update Prometheus remote protocol. <!-- https://github.com/influxdata/influxdb/pull/17814 -->
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:37]: read, write or both??

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 5]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
-- Support authorization for Kapacitor via LDAP.
+  Allow for collection of statistics of points, values, and new series written per measurement and by login.
+  This data is collected and exposed at the data node level.
+  With these metrics you can, for example:
+  aggregate the write requests across the entire cluster,
+  monitor the growth of series within a measurement,
+  and track what user credentials are being used to write data.
+- Support authentication for Kapacitor via LDAP.
 - Support for [configuring Flux query resource usage](/enterprise_influxdb/v1.9/administration/config-data-nodes/#flux-controller) (concurrency, memory, etc.).
 - Upgrade to [Flux v0.113.0](/influxdb/cloud/reference/release-notes/flux/#v01130-2021-04-21).
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:36]: should we use OSS or Cloud? for this link?  I think we should use the OSS one.

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md (hunk 4 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:37]: @jacobmarble Can you shed some light on this? Here's the relevant PR: https://github.com/influxdata/influxdb/pull/17814.

Comment (timhallinflux) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:37]: looks like Read....

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 6]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
   Node labels let you assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
 - Improve performance of `SHOW SERIES CARDINALITY` and `SHOW SERIES CARDINALITY from <measurement>` InfluxQL queries.
   These queries now return a `cardinality estimation` column header where before they returned `count`.
-- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
+- Improve diagnostics for license problems.
+  Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
 - Add improved [ingress metrics](/enterprise_influxdb/v1.9/administration/config-data-nodes/#ingress-metric-by-measurement-enabled--false) to track points written by measurement and by login.
-- Support authorization for Kapacitor via LDAP.
+  Allow for collection of statistics of points, values, and new series written per measurement and by login.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:28]:
```suggestion
  Allow for collection of statistics regarding points, values, and new series written per measurement and by login.
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 7]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
 - Remove extraneous nil check from points writer.
 - Ensure a newline is printed after a successful copy during [restoration](/enterprise_influxdb/v1.9/administration/backup-and-restore/).
 - Make `entropy show` expiration times consistent with `show-shards`.
+- Properly shutdown multiple HTTP servers.
+- Allow CORS in v2 compatibility endpoints.
+- Address staticcheck warnings SA4006, ST1006, S1039, and S1020.
+- Fix Anti-Entropy looping endlessly with empty shard.
+- Disable MergeFiltersRule until it is more stable.
+- Fix data race and validation in cache ring.
+- Return error on nonexistent shard ID.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:56]:
```suggestion
- Return error on non-existent shard ID.
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 8]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
+- Add `User-Agent` to allowed CORS headers.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:57]:
```suggestion
- Add `User-Agent` to allowed CORS headers.
- Allow CORS in v2 compatibility endpoints.
- Address staticcheck warnings SA4006, ST1006, S1039, and S1020.
- Fix Anti-Entropy looping endlessly with empty shard.
- Disable MergeFiltersRule until it is more stable.
- Fix data race and validation in cache ring.
- Return error on nonexistent shard ID.
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 9]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
+- Fix key collisions when serializing `/debug/vars`.
+- Fix temporary directory search bug.
+- Grow tag index buffer if needed.
+- Use native type for summation in new meancount iterator.
+- Fix consistent error for missing shard.
+- Properly read payload in `snapshotter`.
+- Fix help text for `influx_inspect`.
+- Allow `PATCH` in CORS.
+- Fix `GROUP BY` returning multiple results per group in some circumstances.
+- Add option to authenticate prometheus remote read.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:68]:
```suggestion
- Add option to authenticate Prometheus remote read.
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 10]
```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
...
+- Fix FGA enablement.
+- Fix "snapshot in progress" error during backup.
+- Fix cursor requests (`[start, stop]` instead of `[start, stop)`).
+- Exclude stop time from array cursors.
+- Fix Flux regression in buckets query.
+- Fix redundant registration for prometheus collector metrics.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:74]:
```suggestion
- Fix redundant registration for Prometheus collector metrics.
```

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md (hunk 4 above)
Comment (jacobmarble) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md]: It updates the Prometheus dependency, in order to add streamed reads. Not a feature I'm familiar with, but I tested the regular /metrics endpoint to ensure that the Prometheus exposition data looked sane.

**Pull request merged**